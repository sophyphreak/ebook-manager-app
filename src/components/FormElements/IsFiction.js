import React, { Component } from 'react';

const IsFiction = (props) => (
  <label>
    Fiction or Non-fiction?*
    <input type="text" value={props.value} onChange={props.onChange} />
  </label>
);

export default IsFiction;