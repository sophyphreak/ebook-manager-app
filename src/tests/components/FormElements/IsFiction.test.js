import React from 'react';
import { shallow } from 'enzyme';
import IsFiction from '../../../components/FormElements/IsFiction';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<IsFiction />);
});

test('should render IsFiction correctly', () => {
  expect(wrapper).toMatchSnapshot();
});