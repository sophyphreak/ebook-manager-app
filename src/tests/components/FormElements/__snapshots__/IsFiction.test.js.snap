// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render IsFiction correctly 1`] = `
<RadioHorizontal
  label=""
  options={
    Array [
      "Fiction",
      "Non-fiction",
    ]
  }
/>
`;
